AUTHORS

```diff
@@ -3139,5 +3139,188 @@ Shigma <33423008+Shigma@users.noreply.github.com>
+Dan Čermák <dcermak@suse.com>
+ttzztztz <ttzztztz@outlook.com>
+Vít Ondruch <vondruch@redhat.com>
+humanwebpl <58517331+humanwebpl@users.noreply.github.com>
+Dawid Rusnak <dawid@drcode.pl>
+obi-el <elobiobinna@gmail.com>
+Merlin Luntke <22600241+Luntke@users.noreply.github.com>
+Marko Kaznovac <kaznovac@users.noreply.github.com>
+Gabriel Schulhof <gabrielschulhof@gmail.com>
+Ian Kerins <ianskerins@gmail.com>
+dbachko <dbachko@gmail.com>
+Mattias Buelens <mattias@buelens.com>
+Dylan Elliott <dylanelliott27@hotmail.com>
+Wassim Chegham <github@wassim.dev>
+simov <simeonvelichkov@gmail.com>
+wwwzbwcom <zbwhome@outlook.com>
+Stephen Belanger <stephen.belanger@datadoghq.com>
+David Glasser <glasser@davidglasser.net>
+pezhmanparsaee <p.parsaee@gmail.com>
+Hassaan Pasha <pasha.hassaan@gmail.com>
+Darkripper214 <phakorn214@gmail.com>
+Anu Pasumarthy <anupama.pasumarthy@gmail.com>
+HiroyukiYagihashi <hey@hiroy.uk>
+Arkerone <shaita.axel@gmail.com>
+Voltrex <62040526+VoltrexMaster@users.noreply.github.com>
+ycjcl868 <chaolinjin@gmail.com>
+Serkan Özel <serkanozel380@gmail.com>
+Ferdi <iguessitsokaythen@gmail.com>
+eladkeyshawn <elad.keyshawn@gmail.com>
+luyahan <yahan@iscas.ac.cn>
+Simon Knott <info@simonknott.de>
+Siddharth <siddharth.muscat@gmail.com>
+Cactysman <git@cactys.dev>
+David Brownman <beamneocube@gmail.com>
+Michael Rommel <rommel@layer-7.net>
+Chengzhong Wu <chengzhong.wcz@alibaba-inc.com>
+Andres <and.atencio@gmail.com>
+Jayden Seric <me@jaydenseric.com>
+divlo <contact@divlo.fr>
+Rohit Gohri <work@rohit.page>
+Giora Guttsait <giora111@gmail.com>
+takayama <takayama@foxmail.com>
+Rafael Gonzaga <rafael.nunu@hotmail.com>
+Arnold Zokas <arnold.zokas@coderoom.net>
+Nils Dralle <nils.91@googlemail.com>
+Jesse Chan <jc@linux.com>
+helloyou2012 <helloyou2012@yeah.net>
+MrJithil <jithil@outlook.com>
+Rodolfo Carvalho <rhcarvalho@gmail.com>
+Jordan Baczuk <JBaczuk@users.noreply.github.com>
+moander <moander@outlook.com>
+Hitesh Sharma <hitesh9sharma@gmail.com>
+Andreas Schwab <schwab@suse.de>
+Moritz Kneilmann <hi@kneilmann.tech>
+fisker Cheung <lionkay@gmail.com>
+Issam E. Maghni <issam.e.maghni@mailbox.org>
+TodorTotev <51530311+TodorTotev@users.noreply.github.com>
+Wael Almattar <waelsy123@gmail.com>
+yotamselementor <83912471+yotamselementor@users.noreply.github.com>
+pengjie <37610029@qq.com>
+Philip <phershbe@gmail.com>
+julianjany <54538266+julianjany@users.noreply.github.com>
+bl-ue <54780737+bl-ue@users.noreply.github.com>
+npm-robot <ops+robot@npmjs.com>
+Shaun Keys <mariobuddy@gmail.com>
+bl-ue <bl-ue@users.noreply.github.com>
+Simone Busoli <simone.busoli@nearform.com>
+ycjcl868 <45808948@qq.com>
+Qingyu Deng <i@ayase-lab.com>
+Derevianchenko Maksym <32910350+maks-white@users.noreply.github.com>
+RA80533 <32469082+RA80533@users.noreply.github.com>
+Mao Wtm <m@maowtm.org>
+Voltrex <mohammadkeyvanzade94@gmail.com>
+Houssem Chebab <houssemchebeb@gmail.com>
+Joyee Cheung <joyee@igalia.com>
```

Comment:
@joyeecheung Which email would you like in the AUTHORS file? joyeec9h3@gmail.com or joyee@igalia.com (or something else)?

Reply:
I'd go with the gmail one, the igalia one probably was a mistake I made on my new laptop